frontend/server/src/Controllers/User.php

```diff
@@ -3979,7 +3979,7 @@ public static function getUserDependentsForTypeScript(\OmegaUp\Request $r) {
                     'omegaUpTitleMyDependents'
                 )
             ],
-            'entrypoint' => 'users_rank', // FIXME: Update this to the right entrypoint
+            'entrypoint' => 'user_dependents', // FIXME: Update this to the right entrypoint
```

Comment:
```suggestion
            'entrypoint' => 'user_dependents',
```
Ese "FIXME" ya puede irse, porque era para indicar que se tenía que cambiar acorde una vez hagas precisamente este cambio.